### ISSUE TITLE
Error in PredictCost() for the op: op: "Softmax" attr 

### ISSUE THREAD

**Issue by krishnanNuance** [onnx/onnx-tensorflow#1044]
**Describe the bug**

The model converted from ONNX to TF on validation returns NaN as result.

**To Reproduce**

Please give us instructions to reproduce your problem.

1. export fairseq's transformer lm model to onnx
2.  export the onnx model to TF
3. load the saved_model and validate with any src_token value
4. the output array contains NaN

**ONNX model file**

If applicable, attach the onnx model file in question using Gist, DropBox or Google Drive.

**Python, ONNX, ONNX-TF, Tensorflow version**

This section can be obtained by running `get_version.py` from util folder.
 - Python version: 1.11.0+cu102
 - ONNX version: 1.10.2
 - ONNX-TF version: 1.10.0 + implementation for squeeze and unsqueeze for opset v 13
 - Tensorflow version:  2.8.0

**Additional context**
